ProjectLighthouse/Helpers/CryptoHelper.cs

```diff
@@ -19,8 +19,7 @@ public static class CryptoHelper
     public static string GenerateAuthToken()
     {
         byte[] bytes = (byte[])GenerateRandomBytes(256);
-
-        return BCryptHash(Sha256Hash(bytes));
+        return Convert.ToBase64String(Encoding.UTF8.GetBytes(BCryptHash(Sha256Hash(bytes))));
```

Comment:
Will this interfere with the game at all, or is this logic separate from game authentication tokens?

Reply:
It will likely cause re-auths for game players, I'll do a quick check to make sure that it won't affect anything persistent

Reply:
I've just made it so that only tokens in emails are base64 encoded